app/src/main/java/org/jellyfin/androidtv/ui/GuideChannelHeader.java

```diff
@@ -45,14 +48,39 @@ private void initComponent(Context context, ChannelInfoDto channel) {
         ((TextView) findViewById(R.id.channelName)).setText(channel.getName());
         ((TextView) findViewById(R.id.channelNumber)).setText(channel.getNumber());
         mChannelImage = (ImageView) findViewById(R.id.channelImage);
+
+        if (fragment != null) {
+            mActivity = fragment;
+            this.setFocusable(focusable);
+            setOnClickListener(new OnClickListener() {
+                @Override
+                public void onClick(View v) {
+                    fragment.switchChannel(channel.getId(), false);
+                }
+            });
+        }
     }
 
     public void loadImage() {
-        Glide.with(mActivity)
+        Glide.with(mContext)
                 .load(ImageUtils.getPrimaryImageUrl(mChannel, get(ApiClient.class)))
                 .override(IMAGE_WIDTH, IMAGE_HEIGHT)
                 .centerInside()
                 .into(mChannelImage);
     }
 
+    @Override
+    protected void onFocusChanged(boolean gainFocus, int direction, Rect previouslyFocusedRect) {
+        super.onFocusChanged(gainFocus, direction, previouslyFocusedRect);
+
+        if (gainFocus) {
+            setBackgroundColor(Utils.getThemeColor(getContext(), android.R.attr.colorAccent));
+
+            mActivity.setSelectedProgram(this);
+        } else {
+            setBackground(getResources().getDrawable(R.drawable.light_border));
+        }
+//        TvApp.getApplication().getLogger().Debug("Focus on " + mProgram.getName() + " was " + (gainFocus ? "gained" : "lost"));
```

Comment:
Either remove, uncomment or add a comment explaining why this line is commented to avoid confusion for other devs

Reply:
I'll pull it. That line was already there. I didn't know why myself.